did-tezos/src/explorer.rs

```diff
@@ -0,0 +1,123 @@
+use anyhow::{anyhow, Result};
+use reqwest;
+use serde::Deserialize;
+use ssi::did::{Service, ServiceEndpoint};
+use ssi::one_or_many::OneOrMany;
+
+const BCD_URL: &str = "https://better-call.dev/";
```

Comment:
Hard coding this URL and/or using it as a default requires trust in this third party and the CA system. Could it instead be passed via a [DID Parameter](https://w3c.github.io/did-core/#did-parameters) / [DID Resolution Input Metadata property](https://w3c.github.io/did-core/#did-resolution-options) and/or environmental variable? For testing, a local HTTP server could be used, like how the DID Resolution HTTP(S) binding is tested [here](https://github.com/spruceid/ssi/blob/main/src/did_resolve.rs#L1280-L1289).

Reply:
True, and as BCD can be deployed by anyone (which could also cover a local blockchain) is makes extra sense. I'd be of the opinion to make that an issue and address it later to release this PR faster. What do you think @wyc ?

Reply:
Yes, sounds good. let’s make sure we’re tracking it in an issue.

Reply:
Opened https://github.com/spruceid/ssi/issues/109